docs/tutorial/rln-keystore-generator.md

```diff
@@ -13,7 +13,7 @@ It is meant to be used to generate and persist a set of valid RLN credentials to
 1. First, we compile the binary
     
     ```bash
-    make -j12 rln-keystore-generator
+    make -j16 wakunode2
```

Comment:
What do -j12 and -j16 mean/do? Looked it up and haven't found 😬

Reply:
number of threads for compilation afaik